go/protocols/capture/push_server.go

```diff
@@ -31,19 +34,19 @@ func NewPushServer(
 		return nil, err
 	}
 
-	var combiners [2][]pf.Combiner
```

Comment:
@jgraettinger As far as I could tell, these combiners were being created and then thrown away, I couldn't find anywhere they were being used. Is there some kind of side-effect of creating a combiner that I'm missing?

Reply:
@jshearer I think this was vestigial. Combiner creation happens in the [capture coordinator](https://github.com/estuary/flow/blob/5da9656db62189b74e6935c975d7615ae3f6b34c/go/protocols/capture/coordinator.go#L68), so I think it was ok to remove this.